testsuite/features/support/constants.rb

```diff
@@ -1410,13 +1458,50 @@
   'el9-manager-tools-updates-x86_64-ol9' => 120,
   'el9-manager-tools-updates-x86_64-rocky' => 120,
   'el9-pool-x86_64' => 60,
+  'fake-base-channel-appstream' => 360,
   'fake-base-channel-debian-like' => 300,
   'fake-base-channel-rh-like' => 360,
-  'fake-base-channel-appstream' => 360,
   'fake-child-channel-i586' => 300,
   'fake-child-channel-suse-like' => 240,
   'fake-rpm-suse-channel' => 120,
   'fake-rpm-terminal-channel' => 360,
+  'managertools-beta-el9-pool-x86_64' => 60,
+  'managertools-beta-el9-updates-x86_64' => 60,
+  'managertools-beta-sle15-pool-x86_64-sp4' => 60,
+  'managertools-beta-sle15-updates-x86_64-sp4' => 60,
+  'managertools-beta-ubuntu2204-updates-amd64' => 60,
+  'managertools-beta-ubuntu2404-updates-amd64' => 60,
+  'managertools-debian12-beta-updates-amd64' => 60,
+  'managertools-debian12-updates-amd64' => 60,
```

Comment:
shouldn't we also have the new MLM debian 12 client tools for both amd64 AND arm64 because of the tech preview support for Raspberry Pi OS 12?

Reply:
No we don't use them in the testsuite so we don't add them